.evergreen/functions.yml

```diff
@@ -37,6 +37,10 @@ variables:
     EVERGREEN_TASK_URL: https://evergreen.mongodb.com/task/${task_id}
     EVERGREEN_VERSION_ID: ${version_id}
     EVERGREEN_WORKDIR: ${workdir}
+    # WARN: This version is behind our electron runtime, but updating it will
+    # drop support for some older linux platforms, so we are keeping them out of
+    # sync for now
+    # TODO: https://jira.mongodb.org/browse/COMPASS-7188
```

Comment:
https://jira.mongodb.org/browse/COMPASS-6915 😄

Reply:
Thanks, couldn't find it!